test/run-openshift-remote-cluster

```diff
@@ -24,13 +24,13 @@ ct_os_set_ocp4
 
 trap ct_os_cleanup EXIT SIGINT
 
-ct_os_check_compulsory_vars
+ct_os_check_compulsory_vars || exit $OC_ERR
```

Comment:
Where is the new variable defined?

Reply:
It has been added to `test-lib-remote-openshift.sh` recently:

https://github.com/sclorg/container-common-scripts/commit/85f06416bf48f5c66766e605047d81e97d73199f